validating_locally.md

```diff
@@ -0,0 +1,22 @@
+## Validating `terraform plan` locally (beta)
```

Comment:
Maybe work in progress instead of beta? I'm on the fence about running terraform tasks locally beyond `terraform validate` due to the visibility of state. Or is that what this file is doing and it just needs to be captured more clearly?

Reply:
This file was extracted from the original `README.md`, but I'm happy to change that text.